daprdocs/content/en/developing-applications/building-blocks/jobs/jobs-overview.md

```diff
@@ -57,7 +57,9 @@ The jobs API provides several features to make it easy for you to schedule jobs.
 
 ### Schedule jobs across multiple replicas
 
-The Scheduler service enables the scheduling of jobs to scale across multiple replicas, while guaranteeing that a job is only triggered by 1 scheduler service instance.
+When you create a job, it will replace any existing job with the same name. This means that every time a job is created, it resets the count and only keeps 1 record in the embedded etcd for that job. Therefore, you don't need to worry about multiple jobs being created and firing off — only the most recent job will be recorded and executed, even if all your apps schedule the same job on startup. 
+
+The Scheduler service enables the scheduling of jobs to scale across multiple replicas, while guaranteeing that a job is only triggered by 1 Scheduler service instance. 
```

Comment:
Jobs have a guarantee of at least once.

We also need to include a section about Schedulers being rescheduled or restarted is highly disruptive of  inflight jobs, likely causing them to fire duplicate times.

We need to make it clear that the user should add a priority class to the Dapr control plane which is higher than their applications in order to prevent disruption.

Reply:
The at least once is noted on line 23 above in this same file. I will ensure to note the priority class 👍🏻 thanks for the call out there - Ill probably note that in the Scheduler docs pg tho

Reply:
For the priority class I would consider adding this into here https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-production/#setting-cluster-critical-priority-class-name-for-control-plane-services as the place that people look.